packages/data-stores/src/wpcom-plans-ui/reducer.ts

```diff
@@ -25,9 +26,31 @@ const selectedStorageOptionForPlans: Reducer<
 	return state;
 };
 
+const comparisonGridCompactViewPlanDisplayOrder: Reducer<
```

Comment:
I think we should ditch `CompactView` from the name altogether. The comparison grid has an order irrespective of the grid size, right? From my understanding, it's only currently modifiable in a "compact view" (which in itself is ambiguous - there's no such concept baked into the grid - there's "small", "medium", etc.). I don't think this should be made into the state name. it's just a layout consideration. 

Assuming we go with this, I'd rename to `comparisonGridVisibleGridPlansOrder` (to also be consistent with the how we name these in the component). However, I'd also like us to revisit the approach and try and remove `Order` from names i.e. why not just save here `comparisonGridVisibleGridPlans`?